.vscode/settings.json

```diff
@@ -3,9 +3,14 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.codeActionsOnSave": {
     "source.fixAll": false,
-    "source.fixAll.eslint": false
+    "source.fixAll.eslint": true
   },
   "npm.packageManager": "pnpm",
+  "eslint.options": {
+    "rules": {
+      "comma-dangle": "off"
+    }
+  },
```

Comment:
It's not already defined in ``@prestashopcorp/eslint-config-ts`` ?

Reply:
ok, deletion done